### ISSUE TITLE
Support for Comboot

### ISSUE THREAD

**Issue by vpontin** [satanasov/postlove#27]
Hi all!

Just wonder if you guys would consider adding style files specifically for [Comboot](https://comboot.io/), a style made entirely using Bootstrap 3 (and its icons by Glyphicons and Font Awesome).

I asked this because, unlike other styles, Comboot is deeply different than prosilver in code and layout structure. In fact, it perform better on mobile than the original prosilver.

Here is the [repo](https://github.com/ComBoot/ComBoot-3.1) link. Sadly @TheZoker (the author) wants privacy until he releases the first public version (anyway the theme is already well stable), but access can be easily given via this [page](https://comboot.io/blog/beta/beta-3-1/) (entering github email is enough).

I consider make that compatibility myself, but i actually have no time.

I think it can be release natively within this repo, or made in a [compatibility pack](https://comboot.io/compatibility/) format (which can be free :smile:  or paid :cry: )

Anyway, thanks for reading!

**Comment by satanasov** [comment]
I could probably write such a theme ... I will need the beta version thou ...

**Comment by vpontin** [comment]
@TheZoker what can you say about?

**Comment by vpontin** [comment]
@TheZoker what can you say about?

**Comment by satanasov** [comment]
I have subscribed and I have access to the theme ... I will try working with the beta tonight.

**Comment by TheZoker** [comment]
@satanasov Ok great, let me know, if you need any support :)

**Comment by satanasov** [comment]
If you have all style events, it will be ok.

**Comment by TheZoker** [comment]
Yes ComBoot includes all events listed here: https://wiki.phpbb.com/Event_List#Template_Events

**Comment by satanasov** [comment]
Perfect ... if not tonight or tomorrow's night I'll make the conversion.

**Comment by satanasov** [comment]
A quick question - how should the template name be - ComBoot or comboot ?

**Comment by TheZoker** [comment]
Please name it `comboot`, as described in this entry:
https://comboot.io/blog/support/kb/create-compatibility-pack/

**Comment by TheZoker** [comment]
Do you have a package ready, so I can test it?

**Comment by satanasov** [comment]
Not for the moment - Friday probably ...

**Comment by JacobOre** [comment]
Any updates, would love to use this extension with comboot?

**Comment by satanasov** [comment]
I will submit it for test soon (it's done but I have some other stuff to do).

**Comment by satanasov** [comment]
So I think I have submitted it ...

**Comment by TheZoker** [comment]
Yes i received it :)
I will check it today

**Comment by vpontin** [comment]
Before i close the issue, @TheZoker , where are the pack @satanasov made on the download page? i just see your (paid) packs and Board Rules, OneAll and QuickReply on 3rd party (free) section.

**Comment by JacobOre** [comment]
Any updates?

**Comment by TheZoker** [comment]
Will make it available tomorrow

**Comment by TheZoker** [comment]
@satanasov Did you see my commit?

**Comment by JacobOre** [comment]
..?

**Comment by satanasov** [comment]
By some strange reason I wasn't notified. I'll accept it.

**Comment by JacobOre** [comment]
Where can I find the extension?

**Comment by satanasov** [comment]
I don't understand the question.

**Comment by JacobOre** [comment]
Well, where do I download the extension that has support for ComBoot?

**Comment by satanasov** [comment]
This repo?

**Comment by vpontin** [comment]
He says about the files for Comboot support. I didnt find any 'comboot' folder in "styles" folder. Do you make a compatibility pack?

**Comment by TheZoker** [comment]
@satanasov Sadly, the extensions does not 100% work with the "comboot-way-change". Can you please have a look? I guess it's a issue with your javascript, which I can not change, because its not in the theme folder. 
 I would also suggest, that you use fontawesome instead of the heart image. phpBB will add support for fontawesome in 3.2, so it's a good time to add it. What do you think?

@vpontin I did not release it yet, because it does not 100% work (as described above). If you want to download it, you have to go to the "comboot" branche: https://github.com/satanasov/postlove/tree/comboot

**Comment by JacobOre** [comment]
Thanks for the update; much appreciated.

**Comment by vpontin** [comment]
@TheZoker oh thanks for make me knows, i'm a noob in GIT rules, so i didint notice the branches
